community/events/intro.md

```diff
@@ -0,0 +1,52 @@
+
+:::{admonition} Team checkin prior to an event
+:class: note
+
+Staff should also have a short check-in prior to an event to ensure that all documents, and elements needed for the event are shared, in the right place, etc.
+
+Because the internet might be an issue sometimes at an event, whomever is leading the event should always bring a pen and paper.
+:::
+
+## Event social media and graphics
+
+The community manager will manage online “coms” and social media during an event. However it will be important for whomever is attending the event in person to be in asynchronous communication via the pyOpenSci Slack about key activities, findings, new cool things, etc to be posting about, promoting and highlighting.
+
+In some cases an event might “pop up” last minute - such as an Open Space at pyCon. In this case, the person attending the event may want to modify and reuse graphics posted on social media to publicize the event.
+
+To support this case, we should also create an event folder in our online graphics platform, Canva. This event specific folder will be where we store all graphics used to promote an event both before during and after in one place. This will make it easier for the person at the event to make a quick change to a graphic, add a room number or a time for a pop up event or change in plans, as needed.
+
+:::{admonition} Share the Canva folder with the pyopensci-canva-team
+:class: important
+
+Be sure to share the Canva folder with the `pyOpensci-canva-team` on Canva to ensure that the entire team has edit access to the graphics and can make last minute changes as needed!
+Please also be sure to name files using expressive words that make them easy to find / use / reuse.
+:::
+
+### Preparation prior to an event
+
+Prior to an, in-person sprint, you should have the following things created:
+
+1. A template "sign up" form with data fields that we collect from participants. This allows us to track who attends and event and to followup with them after (if they wish to have communication with us after)
+1. A tabletop “card” that says pyOpenSci. You will need 2-3 cards on hand for any event in case participants are spread across a few tables. This card will be important for events like sprints, workshops and open spaces where pyOpenSci has one or more tables in a large room. It will signal to contributors that we are there and help people quickly find us.
+ * The card should have a qr code that is dynamic (so we can update the url that it points to and reuse the cards). This will allow us to have participants scan the code using their phones, and add their names as participants in the event. Following the event we can then send a thank message to each participant using a mail-merge system.
+
+:::{todo}
+will this work for events as we will want an event name associated with it but it would be annoying to make a new card for every event? UNLESS we make a bunch at once for all events for the year?
```

Comment:
good point. my thought process here was more around tracking the event that we gain participants from.

from a conversion standpoint (if i can try to use that language correctly 😆 ) it would be how many sprints did we have at pycon us 24 and then we could in 5 years have - over 5 years pycon has brought us 100 new contributors but we may find that scipy is only 24 for example. 

Also sometimes it might be a sprint, others it may be an open space of a bof. 

But if we can have 1 generic "card" design, with a dynamic qr code - then we could have a unique form for each event with the event name / location / type embedded (or something like that?) This would break down for pycon if i held a sprint + an openspace but we could have a dropdown for the type of event in the form. 

i hope that makes sense? i'm just thinking it all through.

Reply:
that makes sense, although we could also just update the form itself to have the event listed, that way we don't need to swap out anything with each event